src/DataTablesBundle.php

```diff
@@ -37,7 +38,7 @@ public function build(ContainerBuilder $container)
     /**
      * {@inheritdoc}
      */
-    public function getContainerExtension()
+    public function getContainerExtension(): ?ExtensionInterface
```

Comment:
this shouldn't be needed since the extension class name follows the Symfony convention and Symfony can automatically handle the class name

@see https://symfony.com/doc/current/bundles/extension.html#creating-an-extension-class